src/screens/OtherProfile/index.js

```diff
@@ -31,18 +31,18 @@ import RenderItem from '../ProfileScreen/elements/RenderItem';
 import ReportUser from '../../components/Blocking/ReportUser';
 import ShareUtils from '../../utils/share';
 import SpecificIssue from '../../components/Blocking/SpecificIssue';
+import StorageUtils from '../../utils/storage';
 import dimen from '../../utils/dimen';
+import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
 import useCreateChat from '../../hooks/screen/useCreateChat';
 import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
-import useOtherProfileScreenHooks from './hooks/useOtherProfileScreenHooks';
 import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
-import {CircleGradient} from '../../components/Karma/CircleGradient';
 import {Context} from '../../context';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {blockUser, unblockUserApi} from '../../service/blocking';
 import {setFollow, setUnFollow} from '../../service/profile';
 import {downVote, upVote} from '../../service/vote';
-import {fonts, normalize} from '../../utils/fonts';
+import {fonts} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been rearranged and a new import `import StorageUtils from '../../utils/storage';` has been added. Ensure that this new import does not introduce any unused dependencies or potential security vulnerabilities.
```diff
+ import StorageUtils from '../../utils/storage';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->